File: Tests/TestData/MultiFileCharacterization/SourceFiles/VbLibrary/ImportCases/ClassImportClass.vb
Imports System.Linq.Enumerable
Imports VbNetStandardLib.OuterClass

Namespace ClassImport
    Class ClassImportClass
        Public Function ClassImportUnQualifiedMember() As IEnumerable(Of String)
            Return Empty(Of string)
        End Function

        Public Sub ClassImportUnQualifiedNestedType()
            Dim ic = New InnerClass()
            ic.TestMethod()
        End Sub
    End Class
End Namespace

File: Tests/TestData/MultiFileCharacterization/VBToCSResults/ConvertWholeSolution/VbLibrary/ImportCases/ClassImportClass.cs
using System.Collections.Generic;
using static System.Linq.Enumerable;
using static VbNetStandardLib.OuterClass;

namespace VbLibrary.ClassImport
{
    class ClassImportClass
    {
        public IEnumerable<string> ClassImportUnQualifiedMember()
        {
            return Empty<string>();
        }

        public void ClassImportUnQualifiedNestedType()
        {
            var ic = new InnerClass();
            ic.TestMethod();
        }
    }
}